Dockerfile

```diff
@@ -114,3 +101,22 @@ RUN adduser \
     user
 USER user
 
+####################################
+# Final executables with entrypoints
+####################################
+FROM base AS leader
+
+# TODO(0xaatif): https://github.com/0xPolygonZero/zk_evm/issues/356
+#                this is bad practice
+COPY .env /
```

Comment:
Should be tested but to remove. We have default values in the code and user can set  their own environment variables/mount `.env` if he wants